lib/Cake/Model/Datasource/Database/Mysql.php

```diff
@@ -895,8 +895,10 @@ public function insertMulti($table, $fields, $values) {
 				$i++;
 			}
 		}
+		$this->_connection->query('SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE="NO_AUTO_VALUE_ON_ZERO"');
```

Comment:
Just that I get it right, doing it in ``Model::save()`` would mean it would be executed on every save, isn't it?